en/middleware/uorb_graph.md

```diff
@@ -3,497 +3,6 @@
-            }
-
-            return x0 > xi + xSize || y0 > yi + ySize ||
-                x1 < xi - xSize || y1 < yi - ySize;
-        }
-
-        function prepare(quad) {
-            if (quad.data) {
-                quad.size = sizes[quad.data.index];
-            } else {
-                quad.size = [0, 0];
-                var i = -1;
-                while (++i < 4) {
-                    if (quad[i] && quad[i].size) {
-                        quad.size[0] = Math.max(quad.size[0], quad[i].size[0]);
-                        quad.size[1] = Math.max(quad.size[1], quad[i].size[1]);
-                    }
-                }
-            }
-        }
-    }
-
-    function xCenter(d) { return d.x + d.vx; }
-    function yCenter(d) { return d.y + d.vy; }
-
-    force.initialize = function (_) {
-        sizes = (nodes = _).map(size);
-        masses = sizes.map(function (d) { return d[0] * d[1] });
-    }
-
-    force.size = function (_) {
-        return (arguments.length
-            ? (size = typeof _ === 'function' ? _ : constant(_), force)
-            : size);
-    }
-
-    force.strength = function (_) {
-        return (arguments.length ? (strength = +_, force) : strength);
-    }
-
-    force.iterations = function (_) {
-        return (arguments.length ? (iterations = +_, force) : iterations);
-    }
-
-    return force;
-}
-
-function boundedBox() {
-    var nodes, sizes;
-    var bounds;
-    var size = constant([0, 0]);
-
-    function force() {
-        var node, size;
-        var xi, x0, x1, yi, y0, y1;
-        var i = -1;
-        while (++i < nodes.length) {
-            node = nodes[i];
-            size = sizes[i];
-            xi = node.x + node.vx;
-            x0 = bounds[0][0] - (xi - size[0]/2);
-            x1 = bounds[1][0] - (xi + size[0]/2);
-            yi = node.y + node.vy;
-            y0 = bounds[0][1] - (yi - size[1]/2);
-            y1 = bounds[1][1] - (yi + size[1]/2);
-            if (x0 > 0 || x1 < 0) {
-                node.x += node.vx;
-                node.vx = -node.vx;
-                if (node.vx < x0) { node.x += x0 - node.vx; }
-                if (node.vx > x1) { node.x += x1 - node.vx; }
-            }
-            if (y0 > 0 || y1 < 0) {
-                node.y += node.vy;
-                node.vy = -node.vy;
-                if (node.vy < y0) { node.vy += y0 - node.vy; }
-                if (node.vy > y1) { node.vy += y1 - node.vy; }
-            }
-        }
-    }
-
-    force.initialize = function (_) {
-        sizes = (nodes = _).map(size);
-    }
-
-    force.bounds = function (_) {
-        return (arguments.length ? (bounds = _, force) : bounds);
-    }
-
-    force.size = function (_) {
-        return (arguments.length
-            ? (size = typeof _ === 'function' ? _ : constant(_), force)
-            : size);
-    }
-
-    return force;
-}
-
-
-function constant(_) {
-    return function () { return _; }
-}
-
-} // initializeGraph()
-
-initializeGraph();
-
-</script>
```

Comment:
Was it intentional that this part was removed?

Reply:
@stmoon This looks like due to an old commit by the build bot. Possibly you didn't sync to the PX4 master before making your changes? 

Are you able to drop any commits you did not intend from the PR?

I added some other comments too.

Reply:
Thank you. I will squash commits after reviewing.

Reply:
I found the old commit by me was merged to master.  I will try to fix it. If not, I will push the request again.

Reply:
@hamishwillee I will push the request again to solve the problem simply (https://github.com/PX4/Devguide/pull/1076).

Sorry for the inconvenience.